### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by Carshy**
The current project is a Ruby on Rails capstone project that is about building a mobile web application that allows users to manage their budget and expenditures: A user can have a list of transactions associated with a category so that they can see how much money they spent and on what.

Using the app, a user can do the following:
- register and log in, so that the data is private to them.
- introduce new transactions associated with a category.
- see the money spent on each category.

### CODE REVIEW

**Comment by Carshy** [comment]
I would kindly like to report that Heroku is not responding to my deployment and that limits me to put the live link currently. Kindly allow me to post the link later once the process is complete. It keeps uploading for hours and throws errors. I will update it soon when I sort it out. Thank you

File: spec/requests/categories_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    it 'should return http success' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'should render correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it "should render page contenting 'Food'" do
+      expect(response.body).to include('DETAILS')
```
Comment (VICTOR-CHUKWU) [inline spec/requests/categories_spec.rb]: - [ ] All tests are failing, kindly fix this and submit as you have done a fantastic job :clap: 
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Carshy , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
## Highlights
- [ ] Linters passed
- [ ] correct Git flow
- [ ] Clean code base


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Carshy** [comment]
All changes made as requested

File: spec/models/category_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
```
Comment (OLIPLICHE) [inline spec/models/category_spec.rb]: - Hi, Please the test still failing. Kindly fix it as requested by the previous review. 
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @Carshy, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- The App is working fine 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Carshy** [comment]
All changes made as requested and the test are now passing. Please note this is last week's project and I have spoken to Microver admins Marco Lizza who has told me to submit for approval so that I am progressed to the next module. Thank you

**Comment by prolajumokeoni** [comment]
Hi @Carshy, kindly add a screenshot of the message you got 👍🏿

**Comment by Carshy** [comment]
Hello @prolajumokeoni as requested attached herein is the screenshot of the email from Marco Lizza (Microverse Admin):
image

**Review by prolajumokeoni — APPROVED** [review]
Hi @Carshy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Readme 🥇 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Deploying your application so users can view your app online, this will make your app more accessible 👍🏿  

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**